src/MusicalScore/Graphical/VexFlow/VexFlowMeasure.ts

```diff
@@ -1238,11 +1247,11 @@ export class VexFlowMeasure extends GraphicalMeasure {
                     const vfse: VexFlowStaffEntry = vexFlowVoiceEntry.parentStaffEntry as VexFlowStaffEntry;
                     if (vfse && vfse.vfClefBefore) {
                         // add clef as NoteSubGroup so that we get modifier layouting
-                        const clefModifier: NoteSubGroup = new NoteSubGroup( [vfse.vfClefBefore] );
+                        const clefModifier: NoteSubGroup = new Vex.Flow.NoteSubGroup( [vfse.vfClefBefore] );
```

Comment:
Probably an error, should use the local `NoteSubGroup` and not the one from `vexflow`

Reply:
Yes, you're importing NoteSubGroup already, no need for `Vex.Flow`, it looks to me.

Reply:
Without Vex.Flow, it is not accepted as constructor. Funny.